### DISCUSSION TITLE
Need help with API key

### DISCUSSION THREAD

**Discussion by iankordonis** [brianpetro/obsidian-smart-connections#733] [General]
Hey guys.. I'm a COMPLETE NOVICE to this field of work, but I use Obsidian quite a bit in my day to day. I have utilized Smart Connections I think since the end of last year sometime? I had it set up correctly with an Open AI API key, but now the key is not working. Usage limit is fine. but I am getting an error that says, "An error occurred. See console logs for details." Tried researching a little, but figured this community could be good help for someone learning how these things work! 

Thanks in advance!

**Comment by brianpetro** [comment]
Thanks for trying Smart Connections 😊 

Screenshot the error in the developer console logs so we can help. 

And if you're using OpenAI, make sure you have prepaid API credits in your account 🌴

**Reply by iankordonis** [reply]
Hey Brian, sorry for reaching out again, I'm very new to all this, but I am still having trouble with the API key connection. Did you get a lead on why I might be having an issue?

**Reply by brianpetro** [reply]
In your screenshot, look for the "console" tab next to elements. That tab is what I need to see to identify the error 🌴

**Reply by iankordonis** [reply]
[OBSIDIAN CONSOLE LOGS.txt](https://github.com/user-attachments/files/16967238/OBSIDIAN.CONSOLE.LOGS.txt)
![CONSOLE 3](https://github.com/user-attachments/assets/10c19bcc-0305-47e1-92c1-ee21879f7318)
![CONSOLE 2](https://github.com/user-attachments/assets/8e16bb3b-a9df-453c-839e-d82d8aa1d0fb)
![CONSOLE 1](https://github.com/user-attachments/assets/77d9a4cf-5cb4-43f7-9a53-7b06d860a5db)


See if thats more what you need

**Reply by brianpetro** [reply]
Yeah, that helps, but doesn't show an api error. What does your chat model settings look like? Which platform and model are selected?

**Reply by iankordonis** [reply]
![image](https://github.com/user-attachments/assets/07a2b912-f688-4525-a2c6-dd4f926986cd)

**Reply by brianpetro** [reply]
Those are the embedding settings. Try clicking the settings icon in the Smart chat window 🌴

**Reply by iankordonis** [reply]
![image](https://github.com/user-attachments/assets/d115118b-16b0-40fd-a657-d581b416a0fc)

**Reply by brianpetro** [reply]
You need to enter an API key in that field labelled OpenAI API Key 🌴

**Reply by iankordonis** [reply]
That feels pretty silly of me. I thought it was located in the embedding settings at some point no? Oh well it works now. Thank you

**Comment by iankordonis** [comment]
![console logs](https://github.com/user-attachments/assets/9d43c3e7-4bca-4ee9-b1b6-38dc6a9cdfec)
Not sure what I am looking at here, but let me know if I'm warm.
